dockstore-webservice/src/main/java/io/swagger/api/impl/ToolsImplCommon.java

```diff
@@ -173,6 +174,7 @@ public static Tool convertEntryToTool(Entry<?, ?> container, DockstoreWebservice
                 processImageDataForToolVersion(castedContainer, (Tag)version, toolVersion);
             }
 
+            // TODO: Parse image in descriptor from AppTool
```

Comment:
Is this TODO still applicable?

Reply:
It is, but this isn't a good place for it. I'll make a ticket instead. But if there's an image being used in the descriptor, we don't ever parse and collect it right now. For workflows, that happens on snapshot and for tools, it's the image the image registered which neither apply to an apptool currently.